client.lua

```diff
@@ -147,13 +169,13 @@ RegisterCommand("drugs-coke", function(source, args, rawCommand)
 	end
 end)
 
-RegisterCommand("drugs-lean", function(source, args, rawCommand)
+RegisterCommand("lean", function(source, args, rawCommand)
 	if #args < 1 then
 	    -- Too low args
 	    TriggerEvent('chat:addMessage', {
 			color = { 255, 0, 0},
 			multiline = true,
-			args = {"WadL Drugs", "^2Correct usage of command is ^*/drugs-lean <amount>"}
+			args = {"WadL Drugs", "^2Correct usage of command is ^*/lean <amount>"}

```

Comment:
Would suggest removing "WadL Drugs" and editing it to be "Drugs" or "System"

Reply:
I like the name how it is, also I think it should be "/drugs-" then "/weed" so people see a list of all the drugs they can do in the command suggestions when they type /drugs and are reminded of all the options.